https://github.com/rockowitz/ddcutil/issues/101

From d4859fe0543f7313e6fb202ee0992e7e27ac1e8f Mon Sep 17 00:00:00 2001
From: "Sanford Rockowitz (/shared/home/rock/dot_gitconfig)"
 <rockowitz@minsoft.com>
Date: Tue, 31 Dec 2019 13:16:44 -0500
Subject: [PATCH] ddc_services.c: - init_ddc_services():   - call to
 init_usb_displays() was not ifteted based on USE_USB   - caused link failure
 if building ddcutil without USB support

---
 src/ddc/ddc_services.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ddc/ddc_services.c b/src/ddc/ddc_services.c
index 8da91131..b242b02b 100644
--- a/src/ddc/ddc_services.c
+++ b/src/ddc/ddc_services.c
@@ -136,7 +136,9 @@ void init_ddc_services() {
    adlshim_initialize();
 
    // usb
+#ifdef USE_USB
    init_usb_displays();
+#endif
 
    // ddc:
    ddc_reset_ddc_stats();
